apps/test/unit/code-studio/initApp/projectTest.js

```diff
@@ -41,3 +43,34 @@ describe('project.getProjectUrl', function () {
     expect(project.getProjectUrl('/view')).to.equal('http://url/view?query');
   });
 });
+
+describe('project.saveThumbnail', () => {
+  const STUB_CHANNEL_ID = 'STUB-CHANNEL-ID';
+  let updateTimestamp;
+
+  beforeEach(() => {
+    updateTimestamp = sinon.stub(header, 'updateTimestamp').callsFake(() => {});
```

Comment:
You can skip the `callsFake(() => {})` as this is the default behavior when using `stub` (as opposed to `spy` which does not modify method behavior).